taskt/Core/Automation/Commands/Text File Commands/WriteTextFileCommand.cs

```diff
@@ -1,75 +1,69 @@
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter or Select the path to the text file.")]
-        [Attributes.PropertyAttributes.SampleUsage("C:\\temp\\myfile.txt or [vTextFilePath]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("File Path")]
+        [InputSpecification("Enter or Select the File Path.")]
+        [SampleUsage(@"C:\temp\myfile.txt || {ProjectPath}\myText.txt || {vTextFilePath}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_FilePath { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the text to be written. [crLF] inserts a newline.")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Indicate the text should be written to files.")]
-        [Attributes.PropertyAttributes.SampleUsage("**[vText]** or **Hello World!**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Text")]
+        [InputSpecification("Indicate the Text to write.")]
+        [SampleUsage("Hello World! || {vText}")]
+        [Remarks("[crLF] inserts a newline.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)]
         public string v_TextToWrite { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please select overwrite option")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Append")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Overwrite")]
-        [Attributes.PropertyAttributes.InputSpecification("Indicate whether this command should append the text to or overwrite all existing text in the file")]
-        [Attributes.PropertyAttributes.SampleUsage("Select from **Append** or **Overwrite**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_Overwrite { get; set; }
+        [PropertyDescription("Overwrite Option")]
+        [PropertyUISelectionOption("Append")]
+        [PropertyUISelectionOption("Overwrite")]
+        [InputSpecification("Indicate whether this command should append the text to or overwrite all existing text " +
+                            "in the file")]
+        [SampleUsage("Select Append or Overwrite")]
```

Comment:
`SampleUsage` for Drop down(s) should be empty.

Reply:
Done.